custom_dc/load_data.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+# Script to process stats using simple stats loader.
+# Defaults
+MODE="customdc"
+OUTPUT_DIR=".data/output_$MODE"
+USAGE="Script to load data into custom dc using simple importer.
+Usage: $(basename $0) [Options]
```

Comment:
What is the basename? The filename?

Reply:
yes, just the filename without the path.

So if user runs `$ custom_dc/load_data.sh -h` it will display
`Usage: load_data.sh...`